vignettes/v2_data.Rmd

```diff
@@ -82,11 +82,45 @@ Please contact us if there are more file formats you want to be supported.
 
 For parsing IgBLAST results process the data with MigMap first.
 
-You can load the data either from a single file, a list of repertoire file paths or from a folder with repertoire files. A single file can be loaded as follows:
+You can load the data either from a single file, a list of repertoire file paths or from a folder with repertoire files. 
```

Comment:
_... data from a single file, a list of repertoire file paths, or from a folder with repertoire files.